package.json

```diff
@@ -30,6 +30,9 @@
     "chai-subset": "^1.3.0",
     "eslint": "^3.2.1",
     "eslint-config-frost-standard": "^3.0.0",
+    "eslint-plugin-mocha": "^4.3.0",
+    "eslint-plugin-promise": "^2.0.1",
+    "eslint-plugin-standard": "^2.0.0",
```

Comment:
I just want to make the point that these are actually peer dependencies of `eslint-config-frost-standard` and are already included as `dependencies` there, so if you do a fresh `npm install` you should get them landing at the right place. I'm not sure it's a good idea to have them defined here too. 

Sometimes, when doing an `npm install <package-name>` I've notice `npm` will put dependencies in the nested `node_modules` instead of keeping it flat, but I think that may have been addressed in more recent `npm` releases, b/c I'm not seeing it anymore in my current `node@5.11.0`/`npm@3.8.6` setup.